demos/quickstart/cdn_loader/.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+    // Use IntelliSense to learn about possible Node.js debug attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Chrome-Debug",
+            "type": "chrome",
+            "request": "launch",
+            "url": "file:///${workspaceRoot}/index.html",
+            "runtimeArgs": [
+                //  Enable local 'file://*' access ---
```

Comment:
These comments should use `/* this syntax */`

Reply:
Why?

Reply:
To avoid having them highlighted in red:
![image](https://user-images.githubusercontent.com/1891935/59865299-aec30e00-9356-11e9-9ba9-05c42add9e6a.png)